src/integer/mat_z/from.rs

```diff
@@ -74,6 +74,22 @@ impl MatZ {
             })
         }
     }
+
+    /// Sets all entries along the diagonal to `1` and all other entries to `0`.
+    ///
+    /// # Example
+    /// ```
+    /// use qfall_math::integer::MatZ;
+    ///
+    /// let mut matrix = MatZ::new(2,3).unwrap();
+    /// matrix.set_one();
+    ///
+    /// let mut identity = MatZ::new(10,10).unwrap();
+    /// identity.set_one();
+    /// ```
+    pub fn set_one(&mut self) {
```

Comment:
As this is a `set` function, it should be in the `set` module. But it is probably a better idea to keep it in the `from` module and rename it to `identity(rows, columns)` as I see few to none use cases for setting a matrix to the identity/ one matrix after it has been set appropriately. Hence, this function would create a new matrix, set it to `one` and output it instead of manipulating it.
A bullet proof solution would be to implement both (in the appropriate modules).